std/assembly/typedarray.ts

```diff
@@ -1260,3 +1304,35 @@ function REVERSE<TArray extends ArrayBufferView, T>(array: TArray): TArray {
   }
   return array;
 }
+
+// @ts-ignore: decorator
+@inline
+function WRAP<TArray extends ArrayBufferView, T>(buffer: ArrayBuffer, byteOffset: i32 = 0, length: i32 = -1): TArray {
+  var bufferByteLength = buffer.byteLength;
+  if (byteOffset < 0 || byteOffset >= bufferByteLength) {
```

Comment:
One neat trick here is to combine both cases into a single unsigned comparison btw since every negative value (msb set) is larger when interpreted as unsigned:

```ts
if (<u32>byteOffset >= <u32>bufferByteLength) {
}
```

Reply:
Nice. Fixed